src/app/modules/analytics/services/analytic.service.ts

```diff
@@ -11,7 +11,6 @@ import { removeOauthTokenFromUrl } from '../../shared/services/logging/url-utils
 import { Capacitor } from '@capacitor/core';
 
 // In order to have IntelliSense on the custom events we need to declare Plausible on the global window element.
-
 // This code comes from https://www.lekoarts.de/garden/how-to-add-plausible-analytics-to-gatsby#:~:text=%23-,TypeScript,-In%20order%20to
```

Comment:
Du trenger kanskje ikke ha med # og det som er etter # i denne linken?

Reply:
hashen her er en anchor point som tar deg til et riktig sted på nettsiden (altså til løsningen som jeg lånet) så at du ikke må scrolle ned og prøve  å finne den. hvis du syns det ikke er nødvending så kan jeg sikkert ta den bort

Reply:
Skjønner, men har du testet linken sjøl?
Jeg tror nok denne er mer riktig: 
https://www.lekoarts.de/garden/how-to-add-plausible-analytics-to-gatsby#typescript